src/components/Auth/Auth.js

```diff
@@ -0,0 +1,113 @@
+import { MDCTextField } from "@material/textfield";
+import { MDCRipple } from "@material/ripple";
+import { MDCTabBar } from "@material/tab-bar";
+
+import { AuthService } from "../../services/AuthService";
+
+import authTemplate from "./Auth.html";
+
+export class AuthComponent {
+  constructor(mountPoint, props = {}) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  querySelectors() {
+    this.email = this.mountPoint.querySelector(".auth__email");
+    this.password = this.mountPoint.querySelectorAll(".auth__password");
+    this.name = this.mountPoint.querySelector(".auth__name");
+    this.next = this.mountPoint.querySelector(".auth__button");
+    this.tabBar = this.mountPoint.querySelector(".auth__tab-bar");
+    this.tabSignIn = this.mountPoint.querySelector(".auth__tab-sign-in");
+    this.tabSignUp = this.mountPoint.querySelector(".auth__tab-sign-up");
+    this.input = this.mountPoint.querySelectorAll(".auth__input");
+    this.erorPoint = this.mountPoint.querySelector(".auth__eror");
+  }
+
+  initMaterial() {
+    this.emailPoint = new MDCTextField(this.email);
+    Array.from(this.password).forEach(el => {
+      this.passwordPoint = new MDCTextField(el);
+    });
+    this.namePoint = new MDCTextField(this.name);
+    this.nextPoint = new MDCRipple(this.next);
+    this.tabBarPoint = new MDCTabBar(this.tabBar);
+  }
+
+  addEventListeners() {
+    this.tabSignIn.addEventListener(
+      "click",
+      this.handleSignInTabClick.bind(this)
+    );
+    this.tabSignUp.addEventListener(
+      "click",
+      this.handleSignUpTabClick.bind(this)
+    );
+    this.next.addEventListener("click", this.handleNextClick.bind(this));
+  }
+
+  handleSignInTabClick() {
+    if (this.tabSignIn.classList.contains("auth__input--active") === false) {
```

Comment:
if (!this.tabSignIn.classList.contains("auth__input--active"))

Reply:
Maksym told us that we shoudn't use ! Because it is hard to read